numba/tests/test_np_functions.py

```diff
@@ -1356,19 +1363,20 @@ def test_triu_indices_from(self):
         self._triangular_indices_from_exceptions(triu_indices_from_arr_k, True)
 
     def partition_sanity_check(self, pyfunc, cfunc, a, kth):
-        # as NumPy uses a different algorithm, we do not expect to match outputs exactly...
+        # as NumPy uses a different algorithm, we do not expect to
+        # match outputs exactly...
         expected = pyfunc(a, kth)
         got = cfunc(a, kth)
 
-        # ... but we do expect the unordered collection of elements up to kth to tie out
+        # but we do expect unordered collection of elements up to kth to tie out
```

Comment:
```suggestion
        # but we do expect the unordered collection of elements up to the kth to tie out
```
this needs more definite article (and probably a line wrap).